src/app/scheduling/daterangepicker/daterangepicker-styling/daterangepicker-styling.scss

```diff
@@ -0,0 +1,49 @@
+// in component.scss
+@import '~igniteui-angular/lib/core/styles/themes/index';
+
+// common
+$purple: #9E379F;
+$blue: #61AEDB;
+
+$custom-palette: igx-palette($primary: $blue, $secondary: $purple);
+
+$today-text: igx-color($custom-palette, "primary", 500);
+$text-color: igx-color($custom-palette, "secondary", 200);
+$color-focused: igx-color($custom-palette, "secondary", 500);
+
+
```

Comment:
I would avoid doing this. Use hardcoded values instead of palettes for brevity's sake.

Reply:
I would prefer keeping the palette, as otherwise it will require up to a half a dozen more properties to be manually set, in order to achieve a similar overall sample look. For reference: https://github.com/IgniteUI/igniteui-angular-samples/blob/vNext/src/app/scheduling/datepicker/datepicker-styling-sample/datepicker-styling-sample.component.scss